flowsa/data_source_scripts/stewiFBS.py

```diff
@@ -36,18 +36,24 @@ def stewicombo_to_sector(yaml_items):
     import stewicombo
     from flowsa.data_source_scripts.EPA_NEI import drop_GHGs
 
-    NAICS_level_value = sector_level_key[yaml_items['NAICS_level']]
+    # determine if fxns specified in FBS method yaml
+    if yaml_load['functions'] != None:
```

Comment:
I think this should be is None

Reply:
oh, I think it should be 
`if 'functions' not in yaml_load:`

Reply:
yes, the above syntax works if 'function' is not in the fbs yaml. 
`if yaml_load['functions'] is None:`
throws an error